ddR/R/ddR.R

```diff
@@ -246,6 +255,8 @@ dmapply <- function(FUN ,..., MoreArgs=list(),
   # Ensure that ... arguments are of equal length. length() works correctly for data.frame,
   # arrays, and lists
   lens <- vapply(dargs,function(x) {
+    # TODO Clark: Why even allow the possibility of this happening where
```

Comment:
Yes, I was thinking of requiring them to restart the R session. @fun-indra if I was doing that I'd definitely be restarting my R session. But not everyone has that workflow, and I agree it's better not to force it on people.